src/OrleansTestingHost/TestingSiloOptions.cs

```diff
@@ -46,6 +50,8 @@ public class TestingSiloOptions
         public bool ParallelStart { get; set; }
         public GlobalConfiguration.ReminderServiceProviderType ReminderServiceType { get; set; }
         public string DataConnectionString { get; set; }
+        public Action<ClusterConfiguration> ConfigurationCustomizer { get; set; }
```

Comment:
Perhaps. FYI: the existing test framework was not suited for multi-cluster testing (TestingSiloHost has many assumptions hardcoded that do not work for multi-cluster scenarios). We chose not to change TestingSiloHost fundamentally because it would make everyone else's life complicated. Instead we are using a different base class, TestingClusterHost. Therefore we cannot use AdjustConfiguration.